ray-operator/hack/local_deploy.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+PROJECT_ROOT="${SCRIPT_DIR}/../.." # hack/ -> ray-operator/ -> kuberay/
+
+HELM_CHART_PATH="${PROJECT_ROOT}/helm-chart/kuberay-operator"
+
```

Comment:
nit: remove this empty line

we can group variable declaration in the same block, no need to add breaks